.github/workflows/static-files-lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Static Files Lint
```

Comment:
Is there a reason to use two different CI instead of two different jobs? (static-files-lint and edison-CI)

Reply:
I haven't thought about it until just now, I'll merge this two files.